google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/ExternalTableDefinition.java

```diff
@@ -212,6 +215,13 @@ public Boolean ignoreUnknownValues() {
   @Nullable
   public abstract Boolean getAutodetect();
 
+  /**
+   * [Experimental] Returns the HivePartitioningOptions when the data layout follows Hive partitioning convention
+   */
+  @SuppressWarnings("unchecked")
+  @Nullable
+  public abstract HivePartitioningOptions getHivePartitioningOptions();
```

Comment:
this breaks binary compatibility

Reply:
Thanks @stephaniewang526 for the review. I will update the patch and send again.

Reply:
@stephaniewang526, @pmakani I updated the patch and I wonder if the tests would automatically run?

Reply:
@stephaniewang526, thanks for triggering the tests. It was my bad with the previous patch - I missed a line that I fixed locally related to the test failures. Could you please trigger the tests again?

Reply:
Thanks @stephaniewang526. All tests passed now.